packages/webdriverio/src/test.ts

```diff
@@ -723,15 +723,16 @@ describe('@axe-core/webdriverio', () => {
           });
         });
 
-        it('returns an error as the first argument', async () => {
-          await client.url(`${addr}/index.html`);
+        it('returns an error as the first argument', done => {
+          client.url(`${addr}/index.html`);
```

Comment:
You need to wait for this to load. Otherwise this test doesn't prove what you're trying to.

```suggestion
          client.url(`${addr}/index.html`);
          .then(() => 
```

Reply:
test fixed had to add a `// @ts-ignore` due to this error message
```console
TSError: ⨯ Unable to compile TypeScript:
src/test.ts:727:44 - error TS2339: Property 'then' does not exist on type 'void'.

727           client.url(`${addr}/index.html`).then(() => {
```